autode/calculations/executors.py

```diff
@@ -199,7 +199,7 @@ def clean_up(self, force: bool = False, everything: bool = False) -> None:
 
         logger.info(f"Deleting: {set(filenames)}")
 
-        for filename in set(filenames):
+        for filename in [fn for fn in set(filenames) if fn is not None]:
```

Comment:
I think this might not even be necessary. I was looking through the code, and the problem seems to be coming from `self.output.filename` which is `None` if `self.output` is a `BlankCalculationOutput()`. Here:
https://github.com/duartegroup/autodE/blob/cbb807024daa7a15924d6b2a9690a4d899854908/autode/calculations/executors.py#L194-L198

Just doing 
```python
        if everything:
            if not isinstance(self.output, BlankCalculationOutput):
                filenames.append(self.output.filename)
            filenames += [
                fn for fn in os.listdir() if fn.startswith(self.name)
            ]
```
fixes the problem and may be better imho.

Reply:
@t-young31 Ok, I see what you mean.